### ISSUE TITLE
Adjustable Age Restrictions for viewing content per user

### ISSUE THREAD

**Issue by GGProGaming** [Fallenbagel/jellyseerr#501]
### Description

Age restrictions for content people are allowed to see in movies and tv and what they can request. Preferably age restrictions and/or category restrictions that the admin sets on a account, and that account can only see, search, and request content that fits in those parameters.

Half of the front movies page is horror or romance and being able to exclude or select which genres you see would be great.

### Desired Behavior

Adjustable search and page filters based on genre and maturity.

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by dpieski** [comment]
This would be excellent. I was hoping that this was a feature that could be implemented. 

It would also be nice to have an third "default" Radarr/Sonarr so that it is easier to segregate based on age-appropriate media. (e.g., for "kids" movies/tv)

**Comment by skorpeyon** [comment]
I'd like to add support for this request, it would be great to allow my kiddo to get on and request age appropriate things for my Emby server.

**Comment by hydrian** [comment]
I need this feature so much....

**Comment by fallenbagel** [comment]
> I need this feature so much....

#1418

Soon™️

**Comment by madcar86** [comment]
I agree per user content filter would be great!!!

**Comment by joe85** [comment]
I'll add it if @fallenbagel promises to not leave my PR in purgatory like the other 35 outstanding PRs.

**Comment by fallenbagel** [comment]
> I'll add it if @fallenbagel promises to not leave my PR in purgatory like the other 35 outstanding PRs. 

None of the prs are in purgatory. They get merged as work continues.

**Comment by ThermicWaffle** [comment]
Bumping. Any updates on this? V2.6 was helpful with filtering, but I’d still like to set defaults to block adult content from my family.

**Comment by scottjamesrogers** [comment]
Also looking for this. The current functionality exposes unsuitable content for the kiddos, even if I can decline their requests, it's not ideal that they can browse the UI, review R rated titles and click to view trailers etc.

Presume this functionality could leverage the existing content filters by disabling them per user?

**Comment by Shallnan-Hub** [comment]
Checking in on this. I would love to roll this out to the family, but need some parental filters.

**Comment by hadshaw21** [comment]
Just bumping this also as it seems it may have gone stale, but would be a greatly appreciated feature :)

**Comment by BrydonP** [comment]
I'd like to see all Adult Content just blocked by default by config, no reason this needs to be mixed in with everything else...

**Comment by fallenbagel** [comment]
> I'd like to see all Adult Content just blocked by default by config, no reason this needs to be mixed in with everything else...

They already are blocked by default. We literally pass in `includeAdult=false` when we grab stuff from tmdb api. The ones that bypasses are the ones properly not tagged on tmdb. Hence literally blocklist feature was introduced for.
